### PR TITLE
Finished tests and checks

### PR DESCRIPTION
**Pull request by JRafaelSoares**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1,21 +1,33 @@
 from blockchain.blockchain_data_structure import Blockchain
+from Crypto.Hash import SHA256
 
-blockchain = Blockchain("catarina-address", "test", '0.0.0.0', 5000)
+hash_function = SHA256.new()
+
+hash_function.update("catarina-address".encode())
+miner_address = hash_function.hexdigest()
+
+hash_function.update("address1".encode())
```
Comment (CatarinaPedreira) [inline test.py:10]: Why use the same hash here and not a new one for "address1"? I believe using the update() function several times for the same hash accumulates the received contents in that hash but not completely sure

Comment (JRafaelSoares) [reply test.py:10]: Thought it wouldnt, will check!

**Comment by RafaelAPB** [comment]
Nice catch, @CatarinaPedreira. For me, looks good.

**Review by RafaelAPB — DISMISSED** [review]

**Review by CatarinaPedreira — APPROVED** [review]
LGTM!

**Review by RafaelAPB — APPROVED** [review]
LGTM

**Pull request merged**